src/studio/src/designer/backend/Services/Implementation/TextResourceService.cs

```diff
@@ -50,8 +50,8 @@ public async Task UpdateTextResourcesAsync(string org, string app, string shortC
                 {
                     FileSystemObject populatedFile = await _giteaApiWrapper.GetFileAsync(org, app, textResourceFromRepo.Path, shortCommitId);
                     byte[] data = Convert.FromBase64String(populatedFile.Content);
-                    TextResource content = data.Deserialize<TextResource>();
-                    TextResource textResourceStorage = await GetTextResourceFromStorage(org, app, content.Language, environmentModel);
+                    Altinn.Platform.Storage.Interface.Models.TextResource content = data.Deserialize<Altinn.Platform.Storage.Interface.Models.TextResource>();
```

Comment:
Any reason for using full namespace?

Reply:
Updated this to use an alias, since I had to do the same in a few other files now.